lib/linter.js

```diff
@@ -329,6 +328,14 @@ function modifyConfigsFromComments(filename, ast, config, ruleMapper) {
                         [].push.apply(disableDirectives, createDisableDirectives("disable", comment.loc.start, value));
                         break;
 
+                    case "eslint-disable-line":
+                        [].push.apply(disableDirectives, createDisableDirectives("disable-line", comment.loc.start, value));
```

Comment:
Thank you for contributing!

If the block comment is multiline, I hope to ignore it and make a warning.

Reply:
Oh, right! I forgot about this. Do we want to only warn in debug mode or always? My only concern is  is that if we start warning by default then we might generate more warnings for users that currently have `eslint-disable` block comments (that currently don't do anything).

I believe in the TSC meeting we discussed having multiline `eslint-disable` comments not do anything currently but also not warn for this case until our next major release. I think it would be fine to warn if we're in debug mode, though. Thoughts @eslint/eslint-team?

Reply:
Actually, looks like we did decide on having multiline comments not apply but also not warning at the meeting. See the notes from the meeting [here](https://github.com/eslint/tsc-meetings/blob/216e2779058a2090a022edb3b0aa4e39de9e4a5c/notes/2017/2017-11-09.md#support-block-comment-version-of-eslint-disable-line).

Reply:
@kaicataldo so is this fine as is and we can create another PR with the warning about using multi-line comments for disabling a single line?

Reply:
I _think_ the current behavior is incorrect because it applies multiline `eslint-disable-*` directives, which is confusing (which line should it be ignoring? The starting line, the ending line, or both?). 

So I believe @mysticatea is saying that those directives should simply be ignored and not applied. We can't actually warn on it though because that would be a breaking change. If anyone already mistakenly has multiline block `eslint-disable-*` directives (even though they don't do anything at the moment), this would create more warning and fail the linting job.

So I believe the correct behavior here is to not apply mutiline `eslint-disable-*` directives in this PR, but not warn on them. We can add a warning in another PR when we're getting ready for our next major release.

@mysticatea Please correct me if I'm wrong.

Reply:
Yes, it would apply to both `eslint-disable-line` and `eslint-disable-next-line`.

Reply:
Thanks for the clarification @not-an-aardvark